documentation/modules/con-securing-client-authentication.adoc

```diff
@@ -5,21 +5,33 @@
 [id='con-securing-client-authentication-{context}']
 = User authentication
 
+[role="_abstract"]
 User authentication is configured using the `authentication` property in `KafkaUser.spec`.
 The authentication mechanism enabled for the user is specified using the `type` field.
 
-Supported authentication mechanisms:
+Supported authentication types:
 
-* TLS client authentication
-* SCRAM-SHA-512 authentication
+* `tls` for TLS client authentication
+* `tls-external` for TLS client authentication using external certificates
+* `scram-sha-512` for SCRAM-SHA-512 authentication
 
-When no authentication mechanism is specified, the User Operator does not create the user or its credentials.
+If `tls` or `scram-sha-512` is specified, the User Operator creates authentication credentials when it creates the user.
+If `tls-external` is specified, a TLS user is created without authentication credentials.
```

Comment:
I think this is confusing. _Creating the TLS user_ is generating the certificate / authentication credentials. So you cannot really say _user is created_ at the same time as _without authentication credentials_. Maybe the below would be more accurate.

```suggestion
If `tls-external` is specified, the user is expected to use TLS client authentication, but no authentication credentials are created.
```